core/indigo-core/common/math/algebra.h

```diff
@@ -303,6 +303,54 @@ namespace indigo
             _rightTop = other._rightTop;
         }
 
+        inline bool pt_in_rect(const Vec2f& pt) const
```

Comment:
Let's follow one naming style. The file uses a mix of camelCase and Snake case styles but majority of methods follow camelCase.

Reply:
fixed